test/parallel/test-util-inspect.js

```diff
@@ -588,8 +588,9 @@ assert.doesNotThrow(function() {
                      '{ a: 123, inspect: [Function: inspect] }');
 
   const subject = { a: 123, [util.inspect.custom]() { return this; } };
+  const UTC = 'util.inspect.custom';
```

Comment:
IMHO 'UTC' isn't a particularly good name. To me, it makes me think of Coordinated Universal Time, but the variable value does not hold anything date/time-related.